### DISCUSSION TITLE
Mocked loot share for boss drops

### DISCUSSION THREAD

**Discussion by czclaxton** [runelite/runelite#15746] [Ideas]
Are there any plugins to artificially implement or fake loot share notification for boss drops? I do not mean actually changing who the drop is given to but to show a notification or message that would say one of the group members "receives" it. 

For example, player A does 90% of the damage and would continue to always get the drop as normal but with a plugin, player B could "win" the drop through a loot share notification. Player A could then trade the drop to Player B. My Ironman group would love something like this because some of us stronger than the rest. 

One cool feature would be having options that could either scale the drop chance with the damage each player has done to the boss or even just an equal chance between everyone. Is there anything like this?

**Comment by Nightfirecat** [comment]
We're not planning on adding this feature, and I wouldn't recommend anyone try to write this as a hub plugin either, due to the clear possibility for abuse by bad actors.
